paper-input-container.html

```diff
@@ -201,8 +202,8 @@
       .input-content.label-is-floating ::content .paper-input-label {
         -webkit-transform: translateY(-75%) scale(0.75);
         transform: translateY(-75%) scale(0.75);
-        -webkit-transition: -webkit-transform 0.25s;
-        transition: transform 0.25s;
+        -webkit-transition: -webkit-transform 0.25s, width 0.26s;
```

Comment:
Would you mind adding a comment, please, as to why this is needed? Just so future us aren't weirded out by the 0.01s difference?